src/BugsnagUnity/Native/Windows/NativeClient.cs

```diff
@@ -21,6 +21,7 @@ public NativeClient(IConfiguration configuration)
 
         public void PopulateApp(App app)
         {
+            app.AddToPayload("type", "Windows");
```

Comment:
Has this been checked with other teams? My understanding is that this field can affect analytics reports and might require backend changes.

Reply:
Yep - this is the agreed string for Windows (as per Electron: https://github.com/bugsnag/bugsnag-js/blob/next/packages/plugin-electron-app/app.js#L8)